src/WTG.Analyzers.TestFramework/TestData/SampleDataSet.cs

```diff
@@ -129,11 +133,15 @@ static SampleDataSet GetSampleData(string name, string? diagnosticsXml, string?
 					result,
 					ImmutableArray<DiagnosticResult>.Empty,
 					ImmutableHashSet<string>.Empty,
-					ImmutableArray<OSPlatform>.Empty);
+					ImmutableArray<OSPlatform>.Empty,
+					omitAssemblyReferences: false);
 			}
 
 			static LanguageVersion ToLanguageVersion(string? ver)
 				=> LanguageVersionFacts.TryParse(ver, out var tmp) ? tmp : LanguageVersion.Default;
+
+			static bool ToBoolean(string? text)
+				=> bool.TryParse(text, out var tmp) ? tmp : false;
```

Comment:
```csharp
=> bool.TryParse(text, out var tmp) && tmp;
```
:)

Reply:
We really should run Analysers over itself somehow :D